### PR TITLE
feat(angular): support angular 15.1.0

### PR DESCRIPTION
**Pull request by Coly010**
Add support for Angular 15.1.0

### CODE REVIEW

**Comment by Coly010** [comment]
Build is currently failing because NPM Semver ranges do not find the `0.1501-*` versions of `@angular-devkit/architect` for `@nguniversal/builders` causing incompatible types:

image
image

**Comment by Coly010** [comment]
> Build is currently failing because NPM Semver ranges do not find the `0.1501-*` versions of `@angular-devkit/architect` for `@nguniversal/builders` causing incompatible types:
> 
> image image

Resolved this with Yarn Resolutions, everything working as expected. 

We should remove the resolutions before we merge.

Also, this issue might go away altogether when we switch to using require / dynamic imports in the `webpack-server` builder

**Review by FrozenPandaz — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -310,7 +310,8 @@
   "resolutions": {
     "**/xmlhttprequest-ssl": "~1.6.2",
     "minimist": "^1.2.6",
-    "underscore": "^1.12.1"
+    "underscore": "^1.12.1",
+    "@nguniversal/builders/@angular-devkit/architect": "~0.1501.0"
```
Comment (robertIsaac) [inline package.json:314]: isn't that a mistake? @Coly010

Comment (Coly010) [reply package.json:314]: No, until @nguinversal releases a new version, this is required

Comment (Coly010) [reply package.json:314]: which they have now, so i'll create a follow up PR

**Pull request merged**